usr/src/cmd/zpool/zpool_vdev.c

```diff
@@ -561,6 +561,8 @@ get_replication(nvlist_t *nvroot, boolean_t fatal)
 	    &top, &toplevels) == 0);
 
 	lastrep.zprl_type = NULL;
```

Comment:
Looks like the {0} wasn't there when we made this change.  I'll remove these changes since it should work fine with the `= {0}`.